base/applications/mspaint/registry.cpp

```diff
@@ -124,7 +126,7 @@ void RegistrySettings::Load()
         ReadDWORD(text, _T("PointSize"),    PointSize,      FALSE);
         ReadDWORD(text, _T("PositionX"),    FontsPositionX, FALSE);
         ReadDWORD(text, _T("PositionY"),    FontsPositionY, FALSE);
-        ReadDWORD(text, _T("ShowTextTool"), ShowTextTool,   FALSE);
+        ReadDWORD(text, _T("ShowTextTool"), (DWORD&)ShowTextTool, FALSE);
```

Comment:
I suppose that if no typecast, compilation emits an error? Then I think I'll revert my original suggestion, and you may actually keep those as DWORDs instead of BOOL :sweat_smile:
In any case, this shouldn't change the rest of your code (the usages in ShowWindow)

Reply:
`../base/applications/mspaint/registry.cpp:129:45: error: cannot bind non-const lvalue reference of type 'DWORD&' {aka 'long unsigned int&'} to an rvalue of type 'DWORD' {aka 'long unsigned int'}`

Reply:
Reverted.